src/ci/docker/x86_64-gnu-tools/repo.sh

```diff
@@ -60,12 +60,14 @@ commit_toolstate_change() {
     OLDFLAGS="$-"
     set -eu
 
-    git config --global user.email '34210020+rust-toolstate-update@users.noreply.github.com'
+    BRANCH=test
+
+    git config --global user.email '7378925+rust-toolstate-update@users.noreply.github.com'
```

Comment:
s/rust-toolstate-update/rust-highfive/

.... maybe? (unsure)

Reply:
@alexcrichton  The `+` part is just a comment, and 7378925 is already `rust-highfive`'s user ID :(

Reply:
Indeed! I can't think of any other reason though locally why it wouldn't be working...

Maybe I could try regenerating the tokens if this update doesn't work?